sfge/src/test/java/com/salesforce/rules/multiplemassschemalookup/SchemaLookupInAnotherMethodTest.java

```diff
@@ -0,0 +1,58 @@
+package com.salesforce.rules.multiplemassschemalookup;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
```

Comment:
Disabled tests for another PR.

Reply:
Dumb question, but you've already made sure that these currently fail, right?

Reply:
They do fail :)